Source/Core/DiscIO/WIABlob.h

```diff
@@ -34,7 +34,27 @@ enum class WIARVZCompressionType : u32
   Zstd = 5,
 };
 
-std::pair<int, int> GetAllowedCompressionLevels(WIARVZCompressionType compression_type);
+template <bool gui>
+std::pair<int, int> GetAllowedCompressionLevels(WIARVZCompressionType compression_type)
+{
+  switch (compression_type)
+  {
+  case WIARVZCompressionType::Bzip2:
+  case WIARVZCompressionType::LZMA:
+  case WIARVZCompressionType::LZMA2:
+    return {1, 9};
+  case WIARVZCompressionType::Zstd:
+    if (gui)
+      // The actual minimum level can be gotten by calling ZSTD_minCLevel(). However, returning that
+      // would make the UI rather weird, because it is a negative number with very large magnitude.
+      // Note: Level 0 is a special number which means "default level" (level 3 as of this writing).
+      return {1, ZSTD_maxCLevel()};
+    else
+      return {ZSTD_minCLevel(), ZSTD_maxCLevel()};
```

Comment:
```suggestion
    if (gui)
    {
      // The actual minimum level can be gotten by calling ZSTD_minCLevel(). However, returning that
      // would make the UI rather weird, because it is a negative number with very large magnitude.
      // Note: Level 0 is a special number which means "default level" (level 3 as of this writing).
      return {1, ZSTD_maxCLevel()};
    }
    else
    {
      return {ZSTD_minCLevel(), ZSTD_maxCLevel()};
    }
```

Reply:
I'll meet you in the middle by moving the comment out of the if/else block.